contrib_bots/bots/encrypt_bot/tests.py

```diff
@@ -0,0 +1,32 @@
+import encrypt_bot
+
+def test():
+    for cmd, expected_response in sample_conversation():
+        message = {'content': cmd, 'subject': 'foo',
+                   'display_recipient': 'bar'}
+
+        class ClientDummy(object):
+            def __init__(self):
+                self.output = ''
+
+            def send_message(self, params):
+                self.output = params['content']
+        handler = encrypt_bot.EncryptHandler()
+        client_dummy = ClientDummy()
+        handler.handle_message(message, client_dummy, '')
+        if client_dummy.output != expected_response:
+            raise AssertionError('''
+                cmd: %s
+                expected: %s
+                but got : %s
+                ''' % (cmd, expected_response, client_dummy.output))
```

Comment:
@tommyip what is the need for newlines, when I am already using triple single quotes ' ' '.